bitnami/spring-cloud-dataflow/values.yaml

```
@@ -814,9 +814,9 @@ deployer:
   ##
   volumes: {}
   ## @param deployer.environmentVariables Streaming applications environment variables
-  ## RabbitMQ/Kafka envs. Multiple values are comma separated.
+  ## RabbitMQ/Kafka envs.
```

I'd include a small example about how to use this parameter properly:
```suggestion
  ## RabbitMQ/Kafka envs.
  ## e.g:
  ## environmentVariables:
  ##   - SPRING_RABBITMQ_SSL_ENABLED=false
  ##   - ...
```

---

Great!, thanks